models/DataObject/ClassDefinition/Data/Table.php

```diff
@@ -354,7 +354,7 @@ public function getDataFromResource($data, $object = null, $params = [])
     {
         $unserializedData = Serialize::unserialize((string) $data);
 
-        if ($data === null || $unserializedData === null) {
+        if ($data === null || $unserializedData === null || $unserializedData === "") {
```

Comment:
since it is already casted to string on line 355, don't think so null check is required
```suggestion
        if ($data === null || empty($unserializedData)) {
```

Reply:
@dvesh3 But what if the table has 1 column an 1 row with value `0`?

Reply:
data will be null in that case. please see https://github.com/blankse/pimcore/blob/66c89bf84736c0128b982cbfad42cabff2bfe779/models/DataObject/ClassDefinition/Data/Table.php#L455-L467